src/__test__/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapGroupBySettings.test.jsx

```diff
@@ -100,19 +104,29 @@ describe('HeatmapGroupBySettings', () => {
     // Should be rendered.
     expect(component.find('HeatmapGroupBySettings').length).toEqual(1);
 
-    // With two buttons.
-    const buttons = component.find('Radio');
-    expect(buttons.length).toEqual(2);
+    const reorderableList = component.find('ReorderableList');
+    const groupByItems = reorderableList.find('Item');
+
+    // with one ite
+    expect(groupByItems.length).toEqual(1);
+
+    // the item should be samples
+    expect(groupByItems.at(0).childAt(0).childAt(0).text()).toEqual('Sample');
```

Comment:
We should use css selectors to find the correct element, instead of doing `groupByItems.at(0).childAt(0).childAt(0)`. In this way, if we change the positioning or anything in the `groupByItems`, this test is going to fail. In here you can use `find` and specify the actual component you are looking for, then get its text. Take a look how it is done in other tests, there are lots of other examples like `FocusButton.test.jsx` or `DiffExprResults.test.jsx` or `HideButton.test.jsx`

Reply:
Yes I was already using selectors in other parts of the test, but in this case I thought I had no way to access the text in one of the child divs, but apparently with .text() on the parent I can access it directly, so I'll just remove the `childAt`s.

Reply:
sounds good!